src/components/Popover/Popover.tsx

```diff
@@ -16,19 +17,8 @@ export type PopoverProps = ExtractProps<typeof HeadlessPopover> & {
   /**
    * Popover placement options relative to the trigger element.
    */
-  placement?:
-    | 'top-start'
-    | 'top-end'
-    | 'bottom-start'
-    | 'bottom-end'
-    | 'right-start'
-    | 'right-end'
-    | 'left-start'
-    | 'left-end'
-    | 'top'
-    | 'right'
-    | 'bottom'
-    | 'left';
+  placement?: PopperJSOptions['placement'];
```

Comment:
Iirc I had this written out since the popover did some funky things with placement: `auto` or `auto-start` or `auto-end` (sorry forget which)

Reply:
Lemme test this out today. I can restore if it is misbehaving

Reply:
@jinlee93 So the popper JS documentation is a little undercooked for my tastes. It doesn't explain how the modifiers and placement work. The behavior works as documented but by default isn't desirable for popovers like select or menus (it will open to whichever side has the most clear space, meaning the left and right WAY more than we'd want).

I'll create a ticket to use the modifiers for Popper to only filp between top and bottom for the components we want. This would be very desirable so that the popover can open downward or upward (another reason not to call them all dropDOWNs :))

Reply:
That makes sense to me, thanks!